src/main/core/Resources/config/services/library.yml

```diff
@@ -1,3 +1,7 @@
+parameters:
+    claroline.param.geoip_db_path: '%kernel.project_dir%/var/geoip/GeoLite2-City.mmdb'
+    env(MAXMIND_GEOIP_LICENSE_KEY): ''
```

Comment:
move it in `src/main/core/Resources/config/parameters.yml`.

Reply:
We may want a `platform_options` param for `MAXMIND_GEOIP_LICENSE_KEY` instead.

Reply:
Option added